SPECS/systemd/systemd.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Systemd-250
 Name:           systemd
 Version:        250.3
-Release:        9%{?dist}
+Release:        10%{?dist}
```

Comment:
Please also run buddy builds with enabled tests and verify they still pass.

Reply:
I reran buddy builds with tests enabled, both AMD and ARM are still passing